frontends/qt/main.cpp

```diff
@@ -401,7 +405,15 @@ int main(int argc, char *argv[])
 #if defined(_WIN32)
     // Allow existing app to be brought to the foreground. See `view->activateWindow()` above.
     // Without this, on Windows, only the taskbar entry would light up.
+#if QT_VERSION_MAJOR >= 6
```

Comment:
Nit: since we know that we'll be compiling with QT6, can we get rid of the if statements?